src/api/megaprofile-test.js

```diff
@@ -0,0 +1,16 @@
+import { MegaProfile } from '../utilities/MegaProfile';
+
+// Get A User Data. Delete this prior to merging branch into TSGG
+// Do not enable for public use. This is for development/debugging purposes only.
+// Data can be viewed at /api/megaprofile-test
+// -----------------------------------------------------------------------------
+
+export default async function handler(req, res) {
+  try {
+    const mp = new MegaProfile();
+    const { data } = await mp.get();
```

Comment:
I think that the get method should take two parameters. 
1. The endpoint path after all of the prefix and version stuff
2. The encoded SUID, which should be pulled from the logged-in user (JWT in a cookie has this info)

```suggestion
    const { data } = await mp.get('/profiles/contact', user.encodedSUID);
```

With these two params, we can also hit other endpoints like 

```suggestion
    const { data } = await mp.get('/profiles/affiliations', user.encodedSUID);
```